src/fragments/lib-v1/storage/js/getting-started.mdx

```diff
@@ -219,8 +219,11 @@ To make calls to your S3 bucket from your App, you need to set up a CORS Policy
 The following steps will set up your CORS Policy: 
 
 1. Go to [Amazon S3 Console](https://s3.console.aws.amazon.com/s3/home?region=us-east-1) and click on your project's `userfiles` bucket, which is normally named as [Bucket Name][Id]-dev. 
+![Go to [Amazon S3 Console]](/images/storage/CORS1.png)
 2. Click on the **Permissions** tab for your bucket, and then click on the **Cross-origin resource sharing (CORS)** tile.
+![Click on the **Permissions** tab for your bucket](/images/storage/CORS2.png)
```

Comment:
QQ
Is there a "Cross-origin resource sharing (CORS)" title on the CORS2 image.
I think if we scroll down on the Permission tag page this might occur. Should we scroll down and add an image of that. In the description we could mention "Go to the Permissions tab and find the following title"